demo.qmd

```diff
@@ -38,15 +38,15 @@ When you **Render**, a document will be generated that includes both content and
 
 *TODO: day before clinic, make this Python code (don't add screenshot - fewer files to for folks to get distracted with, lighter weight repo)*
 
-```{r}
+```{python}
 2 * 2
 ```
 
 You can add options to executable code. The `echo: false` option disables the printing of code (only output is displayed).
 
 ## Stefanie
 
-For your first edit, you could fix this tpyo and preview how it will look in the site.
+For your first edit, you could fix this typo and preview how it will look in the site.
```

Comment:
```suggestion
For your first edit, you could fix this typo and preview how it will look in the site once it is rendered.
```

This might be a good clarification